contracts/HubPool.sol

```diff
@@ -258,7 +254,17 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         return _exchangeRateCurrent(l1Token);
     }
 
-    function liquidityUtilizationPostRelay(address token, uint256 relayedAmount) public returns (uint256) {}
+    function liquidityUtilizationCurrent(address l1Token) public nonReentrant returns (uint256) {
```

Comment:
Do we use `liquidityUtilizationCurrent` often or is it worth just removing this function?

Reply:
front end uses it. we dont have a byte code issue rn so I think it's fine.

Reply:
ok cool